redhat-developer/app-services-cli
# chore: added a new go-i18n-linter

Closes #1076 

### Verification Steps
1. Run `make lint`
2. Verify the output. As of right now, there should be three strings that doesn't exist in *.toml files

### Type of change


### Checklist


As I already mentioned here https://github.com/redhat-developer/app-services-cli/issues/1076#issuecomment-925750071 the plugin is definitely working once I use it with golangci-lint that I built locally. 

@craicoverflow, @wtrocki I need your help with finding a solution for that.
